test/spec/modules/appnexusBidAdapter_spec.js

```diff
@@ -261,6 +261,8 @@ describe('AppNexusAdapter', function () {
               singleArrNum: [5],
               multiValMixed: ['value1', 2, 'value3'],
               singleValNum: 123,
+              emptyStr: '',
```

Comment:
I am not sure if this is valid use case but what will happen if someone adds `emptyStr: ' '` (space is added as value).

Reply:
the space character is treated as the value for the key-pair and is preserved in the ut request.

eg `"keywords":[{"key":"color","value":["red"]},{"key":"mytest"},{"key":"myspacetest","value":[" "]}]`

Reply:
As a follow-up to the above, the treatment of the space character has not been changed through this PR.  Using a space character as a keyvalue in `master` would result in the same type of object for the ut request as noted above.